third-party/CMakeLists.txt

```diff
@@ -116,8 +116,8 @@ endif()
 
 include(ExternalProject)
 
-set(LIBUV_URL https://github.com/libuv/libuv/archive/v1.12.0.tar.gz)
-set(LIBUV_SHA256 41ce914a88da21d3b07a76023beca57576ca5b376c6ac440c80bc581cbca1250)
+set(LIBUV_URL https://github.com/neovim/libuv/archive/c442232790cbcd82da80356c931a8921dc30811.tar.gz)
+set(LIBUV_SHA256 9c521948228a42ae11f7d61ce3c66588ac658bd49919224f7bd61172b459b4c0)
```

Comment:
This should only be done for `if(WIN32)`. For non-Windows we need to stick to a tagged release (`v1.12.0`, we'll upgrade in a separate PR ... it's nice that CI is passing though!)